migrations/versions/0426_add_bounce_type_columns.py

```diff
@@ -0,0 +1,69 @@
+"""empty message
+
+Revision ID: 0426_add_bounce_type_columns
+Revises: 0425_update_system_templates
+Create Date: 2017-04-25 11:34:43.229494
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "0426_add_bounce_type_columns"
+down_revision = "0425_update_system_templates"
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# option 1
+def upgrade():
+    # 1 - add feedback types to notifications/notifications_history table
+    feedback_types = sa.Enum("hard-bounce", "soft-bounce", name="notification_feedback_types")
+    feedback_types.create(op.get_bind())
+    op.add_column("notifications", sa.Column("feedback_type", feedback_types, nullable=True))
```

Comment:
can we have an index on this? and can you add it concurrently?

Reply:
Yes! I just added a second migration for this: https://github.com/cds-snc/notification-api/pull/1768/files#diff-772bb11b0fb474d256bb93ee066ab549088ef7989abef084cff8edfedd2e3633